crates/nu-command/src/default_context.rs

```diff
@@ -264,6 +264,7 @@ pub fn add_shell_command_context(mut engine_state: EngineState) -> EngineState {
             Term,
             TermSize,
             TermQuery,
+            VersionCheck,
```

Comment:
this should probably be gated behind the `network` feature rather than the `os` feature